akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/TestProbe.scala

```diff
@@ -190,40 +185,53 @@ object TestProbe {
    * is decorated with some fishing details and the test is failed (making it convenient to use this method with a
    * partial function).
    *
-   * @param max Max total time without the fisher function returning `CompleteFishing` before failing
-   *            The timeout is dilated.
+   * @param max Max total time without the fisher function returning `CompleteFishing` before failing.
+   *            The timeout is scaled using the configuration entry "akka.actor.testkit.typed.timefactor".
    * @return The messages accepted in the order they arrived
    */
-  def fishForMessage(max: FiniteDuration, hint: String)(fisher: M ⇒ FishingOutcome): immutable.Seq[M] =
-    fishForMessage_internal(max, hint, fisher)
+  def fishForMessage(max: FiniteDuration, hint: String)(fisher: M ⇒ FishingOutcome): immutable.Seq[M]
 
   /**
    * Same as the other `fishForMessage` but with no hint
    */
-  def fishForMessage(max: FiniteDuration)(fisher: M ⇒ FishingOutcome): immutable.Seq[M] =
-    fishForMessage(max, "")(fisher)
-
-  protected def fishForMessage_internal(max: FiniteDuration, hint: String, fisher: M ⇒ FishingOutcome): immutable.Seq[M]
+  def fishForMessage(max: FiniteDuration)(fisher: M ⇒ FishingOutcome): immutable.Seq[M]
 
   /**
    * Expect the given actor to be stopped or stop within the given timeout or
    * throw an [[AssertionError]].
    */
   def expectTerminated[U](actorRef: ActorRef[U], max: FiniteDuration): Unit
 
+  /**
+   * Expect the given actor to be stopped or stop within the default timeout.
+   */
+  def expectTerminated[U](actorRef: ActorRef[U]): Unit
+
   /**
    * Evaluate the given assert every `interval` until it does not throw an exception and return the
    * result.
    *
    * If the `max` timeout expires the last exception is thrown.
    *
-   * If no timeout is given, take it from the innermost enclosing `within`
-   * block.
+   * Note that the timeout is scaled using the configuration entry "akka.actor.testkit.typed.timefactor".
+   */
+  def awaitAssert[A](a: ⇒ A, max: FiniteDuration, interval: FiniteDuration): A
+
+  /**
+   * Evaluate the given assert every `interval` until it does not throw an exception and return the
+   * result.
+   *
+   * If the default timeout expires the last exception is thrown.
+   */
+  def awaitAssert[A](a: ⇒ A, interval: FiniteDuration): A
```

Comment:
I think this should be max rather than interval?

As previous with default params was `def awaitAssert[A](a: ⇒ A, max: Duration = Duration.Undefined, interval: Duration = 100.millis)`

This is causing `ClusterReceptionistSpec` to fail as it was passing in a 5s max which is now used as an interval so never retried

Reply:
Thanks, fixed